lib/tlm/tlm_interpreter.go

```diff
@@ -79,3 +93,34 @@ func InterpretMeasurementValue(measurement proc.Measurement, data []byte) interf
 		return nil
 	}
 }
+
+func InterpretMeasurementValueString(measurement Measurement, data []byte) string {
+	switch measurement.Type {
+	case "int":
+		if measurement.Unsigned {
+			return fmt.Sprintf("%d", InterpretUnsignedInteger(data, measurement.Endianness))
+		}
+		return fmt.Sprintf("%d", InterpretSignedInteger(data, measurement.Endianness))
+	case "float":
+		return fmt.Sprintf("%f", InterpretFloat(data, measurement.Endianness))
+	default:
+		fmt.Printf("Unsupported type for measurement: %s\n", measurement.Type)
```

Comment:
same thing as above about returning and error rather than printing. especially because the empty string is a valid string. If another piece of code calls this, it can't see the console output to know theres an error, so its only info is that it got a valid string and won't know theres an error

Reply:
Fixed